package.json

```diff
@@ -27,7 +27,7 @@
     "pack": "npm run pack:main && npm run pack:renderer",
     "pack:main": "cross-env NODE_ENV=production webpack --colors --config webpack.main.config.js",
     "pack:renderer": "cross-env NODE_ENV=production webpack --colors --config webpack.renderer.config.js",
-    "test": "npm run test:unit",
+    "test": "npm run lint && npm run test:unit",
```

Comment:
should we remove the linting step in the circleci config then?

Reply:
good point. yes, i think so. i added this in because i was sick of running 'yarn test' and then pushing my PR expecting it to pass.

Reply:
done in latest commit dd23bc6